source/crud/tests/README.md

```diff
@@ -693,3 +695,43 @@ maxTimeMS value of 2000ms for the `explain`.
 
 Obtain the command started event for the explain. Confirm that the top-level explain command should has a `maxTimeMS`
 value of `2000`.
+
+### 15. `MongoClient.bulkWrite` with unacknowledged write concern uses `w:0` for all batches
+
+This test must only be run on 8.0+ servers. This test must be skipped on Atlas Serverless.
+
+If testing with a sharded cluster, only connect to one mongos. This is intended to ensure the `countDocuments` operation
+uses the same connection as the `bulkWrite` to get the correct connection count. (See
+[DRIVERS-2921](https://jira.mongodb.org/browse/DRIVERS-2921)).
+
+Construct a `MongoClient` (referred to as `client`) with
+[command monitoring](../../command-logging-and-monitoring/command-logging-and-monitoring.md) enabled to observe
+CommandStartedEvents. Perform a `hello` command using `client` and record the `maxWriteBatchSize` value contained in the
+response.
+
+Construct a `MongoCollection` (referred to as `coll`) for the collection "db.coll". Drop `coll`.
+
+Use the `create` command to create "db.coll" to workaround [SERVER-95537](https://jira.mongodb.org/browse/SERVER-95537).
+
+Construct the following write model (referred to as `model`):
+
+```javascript
+InsertOne: {
+  "namespace": "db.coll",
+  "document": { "a": "b" }
+}
+```
+
+Construct a list of write models (referred to as `models`) with `model` repeated `maxWriteBatchSize + 1` times.
```

Comment:
Would this test execute faster if it used 5 16MB documents, rather than 100,001 small ones?

Reply:
Good idea. Testing the C driver with a sharded 8.0 cluster locally shows the median of 3 runs:

```
With many small docs to exceed `maxWriteBatchSize`  : 0.65s
With few large docs to exceed `maxMessageSizeBytes` : 0.35s
```

Updated prose test to use few large docs.